onnxscript/evaluator.py

```diff
@@ -346,6 +347,7 @@ def _compute_num_outputs(schema: onnx.defs.OpSchema, *args: Any, **kwargs: Any):
 
 
 _cache_models: dict[Any, ort.InferenceSession] = {}
+cache_ort_session = os.environ.get("CACHE_ORT_SESSION", "1")
```

Comment:
As discussed, we can move this to flags.py:

```python
CACHE_ORT_SESSIONS: bool = os.getenv("CACHE_ORT_SESSIONS", "1") == "1"
```

Reply:
Done.